test/utilities.js

```diff
@@ -264,12 +264,15 @@ describe('utilities', function () {
     // Checking if it's really an instance of an Assertion
     expect(assertion2).to.be.instanceOf(assertionConstructor);
 
-    // Test chaining `.length` after a method to guarantee it is not a function's `length`
+    // Test chaining `.length` after a method to guarantee it's not a function's
+    // `length`. Note: 'instanceof' cannot be used here because the test will
+    // fail in IE 10 due to how addChainableMethod works without __proto__
+    // support. Therefore, test the constructor property of length instead.
     var anAssertion = expect([1, 2, 3]).to.be.an.instanceof(Array);
-    expect(anAssertion.length).to.be.an.instanceof(assertionConstructor);
+    expect(anAssertion.length.constructor).to.equal(assertionConstructor);
```

Comment:
@lucasfcosta I'm actually not sure if the behavior is broken; I think it might've just been the test itself making an assumption that isn't true in legacy browsers.

The reason this assertion was failing in IE 10 is because without `__proto__` support, the `addChainableMethod` utility manually copies the context object's properties onto the `assert` object that's returned by the getter. [Reference](https://github.com/chaijs/chai/blob/6611e1a969a568b8e7996dd407ee8b80fa2f109f/lib/chai/utils/addChainableMethod.js#L95-L104)

This works, it's just that chainable methods aren't an `instanceof assertionConstructor` when using this method of copying properties as opposed to the `assert.__proto__ = Object.create(this);` method that's used when `__proto__` is available. Therefore, I think it's correct behavior that `instanceof assertionConstructor` returns false.